module/VuFind/src/VuFind/Content/ObalkyKnihService.php

```diff
@@ -198,4 +226,41 @@ protected function createLocalIdentifier(string $recordid): ?string
         return empty($this->sigla) ? null :
             $this->sigla . '-' . str_replace('-', '', $recordid);
     }
+
+    /**
+     * Get currently available base URL
+     *
+     * @return string
+     */
+    protected function getBaseUrl(): string
+    {
+        if ($this->checkServersAvailability) {
+            return $this->getAliveUrl();
+        }
+        return '';
```

Comment:
Maybe I'm misreading something, but shouldn't this return the first base URL from the configuration by default, rather than failing if checkServersAvailability is disabled?

Also, you might consider making this a ternary statement for brevity (though that's just a matter of opinion :-) ).

Reply:
Thanks for catching this @demiankatz, that is definitely bug. I'll fix it.

Reply:
Thanks for the fix -- just merged!